### PR TITLE
LPS-131760 Remove persistence from bridged Commerce account services

### PR DESCRIPTION
**Pull request by pei-jung**

### CODE REVIEW

**Comment by pei-jung** [comment]
ci:test:commerce

**Comment by pei-jung** [comment]
ci:test:commerce

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:commerce

**Comment by pei-jung** [comment]
@patriciajeanperez can you check if the test failures in the commerce test suite are related to my changes?

File: modules/apps/commerce/commerce-notification-test/src/testIntegration/java/com/liferay/commerce/notification/test/CommerceOrderStatusNotificationTest.java [hunk 1]
```diff
@@ -319,11 +318,7 @@ private void _checkCommerceNotificationTemplate(
 	private static User _user;
 
 	private CommerceAccount _commerceAccount;
-
-	@DeleteAfterTestRun
```
Comment (drewbrokke) [inline modules/apps/commerce/commerce-notification-test/src/testIntegration/java/com/liferay/commerce/notification/test/CommerceOrderStatusNotificationTest.java:324]: The one thing I'd add with this change is to make sure that the `@DataGuard` is scoped to `METHOD`. `@DeleteAfterTestRun` cleans up after every test method, so we should preserve that expectation.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Look great @pei-jung. Just the one inline suggestion about scoping the DataGuard to method. Otherwise this looks good.

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:test:commerce

**Comment by Sharryxue** [comment]
The failed commerce tests in ci:test:relevant are not related to this pr since they also occasionally failed for the same reason on upstream. Will retrigger the relevant tests.
Will retrigger ci:test:commerce for a better results.

**Comment by Sharryxue** [comment]
ci:test:relevant

**Comment by Sharryxue** [comment]
ci:test:commerce

**Comment by pei-jung** [comment]
ci:test:sf

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**